go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/dapr/dapr
 
-go 1.20
+go 1.21
```

Comment:
Please don't update to Go 1.21 in this PR :) That will need to be done eventually, but let's do that in a separate PR!